### ISSUE TITLE
-m120 -a1 optimized problem

### ISSUE THREAD

**Issue by cagdasb** [hashcat/hashcat#4134]
I have a hash:salt:pepper sha1($salt.$pepper.$pass) 
lets say
```
hash = 58b712c386dc3ed481948494f5541b5f8a644dd8
salt = ABCDEFGHI
pepper = 12345678901234567890123456789012345678901234567890123456
pass= pass
hash:salt.pepper =
90c93e260642ee4a270f030431e9a291a9ff45df:ABCDEFGHI12345678901234567890123456789012345678901234567890123456
```

```
sha1($salt.$pass) 
90c93e260642ee4a270f030431e9a291a9ff45df:ABCDEFGHI12345678901234567890123456789012345678901234567890123456pass
```

I can just assume the salt as salt:pepper and run it. But salt is becoming so long like that (over length 60) so -O doesn't work
So I cut the salt:pepper and make that part length=50
ake the remaining of the salt.pepper as salt2 let's say
```
hash:salt becomes
58b712c386dc3ed481948494f5541b5f8a644dd8:ABCDEFGHI12345678901234567890123456789012345678901

salt2 becomes
234567890123456

pass.txt is 
pass
```

So I should be able to run it as (with OPTIMIZED)
hashcat.exe -m120 -a1 -O hash-salt.txt salt2.txt pass.txt
It runs without no rejected line and DO NOT FIND the hash.

When I run it as (NOT OPTIMIZED)
 hashcat.exe -m120 -a1  hash-salt.txt salt2.txt pass.txt
It DOES FOUND

I hope I am clear on the explanation.
Hashcat version: hashcat-6.2.6+813

**Comment by PenguinKeeper7** [comment]
Yeah, this is unfortunately an issue detailed here: https://github.com/hashcat/hashcat/issues/4067 where the size of the plaintext, combined with the salt is too large for Hashcat to handle and it doesn't reject it, it just simply gets mis-hashed with part of the candidate cut off. The only option currently is to run pure kernels, without using `-O`

**Comment by matrix** [comment]
duplicate of #4067
